rero_ils/modules/patrons/views.py

```diff
@@ -87,6 +88,7 @@ def patron_circulation_informations(patron_pid):
           'engaged': engaged_amount,
           'preview': preview_amount
         },
+        'ill_requests': get_ill_requests_total_for_patron(patron_pid),
         'statistics': get_loans_stats_by_patron_pid(patron_pid),
```

Comment:
maybe 'ill_requests' could be part of the already existing "statistics" attribute ?